shared/src/commonMain/graphql/schema.json

```diff
@@ -338,6 +344,18 @@
             },
             "args": []
           },
+          {
+            "name": "endInstant",
+            "isDeprecated": false,
+            "type": {
+              "kind": "NON_NULL",
+              "ofType": {
+                "kind": "SCALAR",
+                "name": "Instant"
+              }
+            },
+            "args": []
+          },
```

Comment:
Nit: that could use SDL for better readability

Reply:
ah, keep forgetting about that.....this is what I'm using right now.....can't remember, is this different option here?

```
./gradlew downloadApolloSchema --endpoint=https://kiki-conf.ew.r.appspot.com/graphql --schema="shared/src/commonMain/graphql/schema.json"
```

Reply:
I'll  merge this for now and we can do subsequent update for this.

Reply:
If you name your schema `schema.graphqls`, it will be converted to SDL automagically:

```
./gradlew downloadApolloSchema --endpoint=https://kiki-conf.ew.r.appspot.com/graphql 
--schema="shared/src/commonMain/graphql/schema.graphqls"
```